packages/hydrogen/src/customer/customer.ts

```diff
@@ -128,16 +139,48 @@ type CustomerClientOptions = {
   waitUntil?: ExecutionContext['waitUntil'];
   /** This is the route in your app that authorizes the user after logging in. Make sure to call `customer.authorize()` within the loader on this route. It defaults to `/account/authorize`. */
   authUrl?: string;
+  /** This is the route in your app that login() is located.It defaults to `/account/login`. */
+  loginUrl?: string;
+  /** The behaviour when handleUnauthorized, query or mutate is called with an unauthorized customer. The return of method will be throw. The default behaviour is redirecting to `/account/login` where login() is located and pass in current path as redirectPath. */
+  unauthorizedHandler?: () => DataFunctionValue;
```

Comment:
Can we simplify this to only use `unauthorizedHandler`? If they want to change the redirect url they could pass `unauthorizedHandler: () => redirect('/')`?

Reply:
OK, in that case I will remove the loginUrl option!